### DISCUSSION TITLE
What is the name of the file which I will host on live server(for seeing changes I make to css)

### DISCUSSION THREAD

**Discussion by Anshravicode89** [HimanshuNarware/Devlabs#360] [General]
I can't find the file which will be hosted on live server. I am supposed to make changes to look and feel of navbar. Anyone please guide

**Comment by saketh-05** [comment]
@Anshravicode89 Since you want to work on frontend, go to frontend folder (cd frontend) then run required dependencies (npm install). Now you can host on live server (npm start)

**Comment by vaishali-gupta32** [comment]
Open the frontend folder. Open the index.html file and host it on live server.
